lib/client/connection.js

```diff
@@ -56,7 +58,9 @@ function Connection(socket) {
   // state on the agent and read it in middleware
   this.agent = null;
 
+  // TODO: Remove this as a breaking changes in v1.0, and use the 'sharedb' logger and its level
   this.debug = false;
+  logger.setLevel(logger.levels.DEBUG);
```

Comment:
The client side logging appears not configurable at the moment?

Reply:
See above comment about the `debug` flag.